pom.xml

```
@@ -18,13 +18,15 @@
         </license>
     </licenses>
     <scm>
-        <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
-        <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
-        <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
-      <tag>HEAD</tag>
+        <connection>scm:git:git://github.com/${gitHubRepo}.git</connection>
+        <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
+        <url>https://github.com/${gitHubRepo}</url>
+      <tag>${scmTag}</tag>
```

```suggestion
        <tag>${scmTag}</tag>
```

Crappy indentation of XML (also `<xxx/>` ⇒ `<xxx />`) is a longstanding bug of the round-trip library bundled in `maven-release-plugin` used for the two junk commits per release, which would be superseded by this PR.